sharkdevs/Robotics-Django
# #162779276 Create Shell Structure

#### What does this PR do?
* Create a working shell structure for Robotics-Django application.
#### Description of Task to be completed?
* Creates a project directory.
* Creates a Git Repository.
* Creates a Virtual environment.
* Installs dependency of the virtual environment.
* Integrates DRF and API  into the 'Robotics-Django application.

#### How should this be manually tested?
* Create a project folder.
```
$ mkdir projects
```
* Clone the Repo: 
```
$ git clone https://github.com/sharkdevs/Robotics-Django.git 
$ git checkout ch-create-shell-structure-162779276

```
* Create a virtual environment.
```
$ python3 -m virtualenv env

```
* Activate the virtual environment.
```
$ source env/bin/activate
```
* Install dependencies using the 
```
$ pip install -r requirements.txt

```
* Run the server " 
```
$ cd djangorest
$ python3 manage.py runserver

```
* Open your browser on port: http://127.0.0.1:8000/

The screen should reflect as in the screenshot below.

#### Any background context you want to provide?
* n/a
#### What are the relevant pivotal tracker stories?

* https://www.pivotaltracker.com/n/projects/2233385
* id#162779276

#### Screenshots (if appropriate)
<img width="1280" alt="screenshot 2018-12-20 at 04 15 51" src="https://user-images.githubusercontent.com/39020475/50268958-c53aec00-043d-11e9-9ade-bbf1ba1806e1.png">

#### Questions:
